src/helpers/getFullInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import { Person } from "../types";
+
+export const getFullInfo = (peopleFromServer: Person[]) => {
+  return peopleFromServer.map((person) => {
+    const mother = peopleFromServer.find(
+      (mom) => mom.name === person.motherName,
+    );
+    const father = peopleFromServer.find(
+      (dad) => dad.name === person.fatherName,
+    );
```

Comment:
At the time of the search, you don't know if it's dad or mom
```suggestion
    const mother = peopleFromServer.find(({ name }) => name === person.motherName);
    const father = peopleFromServer.find(({ name }) => name === person.fatherName);
```